bumble/device.py

```diff
@@ -808,6 +806,11 @@ def __init__(self, name = None, address = None, config = None, host = None, gene
         # Set the initial host
         self.host = host
 
+    def add_default_services(self, generic_access_service):
```

Comment:
suggest adding a default `True` value for `generic_access_service`, to mirror the default used for the `Device` constructor.